changelog/fragments/3319-deprecate-migrate-subcommand.yaml

```diff
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      The migrate sub-command is deprecated
+    kind: deprecation
+    breaking: true
+    migration:
+      header: The migrate subcommand has been deprecated 
+      body: >
+        ```operator-sdk migrate``` has been deprecated
```

Comment:
Please add a new line here. Also, you would have to run `make generate` and commit the changes, it would update the website docs for this command.